src/App.jsx

```diff
@@ -4,14 +4,26 @@
 import './App.css';
 import DonationForm from './components/DonationForm/DonationForm';
 import DropZone from './components/DropZone/DropZone';
-import uploadImage from './util/furnitureUtils';
-import Dashboard from './pages/dashboard/Dashboard';
-import EditDonationForm from './pages/dashboard/EditDonationForm';
-import Drivers from './pages/dashboard/Drivers';
-import DriverRoutes from './pages/dashboard/DriverRoutes';
+import uploadImage from './utils/furnitureUtils';
+import EditDonationForm from './pages/Dashboard/EditDonationForm';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./pages/Dashboard/EditDonationForm"